recipe/meta.yaml

```diff
@@ -42,6 +42,8 @@ test:
     - morfessor
     - scikit-learn
     - pyemd
+    - mock
+    - boto3
```

Comment:
`boto3` isn't gensim dependency

Reply:
For some reason, the import tests throw an error due to boto3 dependency for smart_open. That's why I added it.